target/linux/ath79/dts/qca9563_dlink_dir-842-c1.dts

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9563_dlink_dir-842.dtsi"
+
+/ {
+	model = "D-Link DIR-842 C1";
+	compatible = "dlink,dir-842-c1", "qca,qca9563";
```

Comment:
Typically compatible is first. Don't know whether that's a rule or just C+P statistics.

Reply:
Changed,thanks!